server/src/main/java/org/apache/iotdb/db/mpp/aggregation/CountAccumulator.java

```diff
@@ -36,28 +37,31 @@ public CountAccumulator() {}
 
   // Column should be like: | Time | Value |
   @Override
-  public int addInput(Column[] column, TimeRange timeRange) {
+  public int addInput(Column[] column, IWindow curWindow) {
+    int windowControlColumnIndex = curWindow.getControlColumnIndex();
```

Comment:
No need to get controlColumnIndex again here, you can assume that ControlColumn has already been put into index 0 by Aggregator.

Reply:
OK. I'll correct.